draft-ietf-rats-architecture.md

```diff
@@ -734,10 +734,49 @@ or interval when changes in operational status, health, and so forth occur.
 
 ## Endorsements
 
-An Endorsement is a secure statement that some entity (e.g., a manufacturer) vouches for the integrity of the
-device's signing capability.  For example, if the signing capability is in hardware, then
-an Endorsement might be a manufacturer certificate that signs a public key whose corresponding
-private key is only known inside the device's hardware.  Thus, when Evidence and such an Endorsement
+Endorsements are essential input to the Verifier that allows them to
+trust Attesters and in turn the Evidence produced by Attesters.
+Without endorsements Verifiers cannot do their job.
```

Comment:
I disagree with both the proposed line 739 and Ned's statement.  Endorsements don't contain "expected values", those are obtained (if at all) in addition to the "endorsement" per se, not as part of them.  (My point is just about terminology/definition, not about the protocol where they may typically come in the same protocol as endorsements.)

Reply:
The Verifier has to have key material. It may also get implied or inferred claims, things it knows about the Attester that the Attester didn't tell it directly. From discussions with Henk and such, I believe both the key material and implied claims come in the form of Endorsements. Some may be known-good-values, but others may be things that are passed on the relying party.